deimos-ruby.gemspec

```diff
@@ -17,29 +17,12 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
+  spec.required_ruby_version = Gem::Requirement.new('>= 2.6.0')
```

Comment:
Unfortunately we need minimum Ruby version to stay at 2.3. We power a legacy app with this framework and we can't upgrade it any time soon.